spec/models/articles_db_view_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ArticlesDbView, type: :model do
+  
+  tag_set = %w(work робота Work Робота)
+  let(:user_data_valid)       do { email: 'admin@example.com' } end
+  let(:article_work_valid)    do { user_id: user_valid.id } end
+  let(:article_invalid_date)  do { user_id: user_valid.id, created_at: (Time.now - 15.days) } end
+  let(:tag_work)              do { name: tag_set.sample } end
+  let(:tag_invalid)           do { name: 'not_a_work' } end
+  let(:article_tag_work)      do { article_id: article.id, tag_id: tag.id } end
+  
+  describe 'ArticleDbView right behaviour with correct tags. ' do
+    subject { described_class.all }
+    let!(:user_valid)   { create(:user, :admin, user_data_valid) }
+    let!(:article)      { create(:article, :published, article_work_valid) }
+    
+    tag_set.map do |tag_item|
```

Comment:
why you need map here? maybe each?

Reply:
`acticle_tag` could be not in this block, put it above

Reply:
Here you are totally right. Both thees thingt are better than my solution